repository_service_tuf/cli/admin/metadata.py

```diff
@@ -628,17 +626,16 @@ def update(
 
 
 def _get_pending_roles(
-    settings: Any, api_url: Optional[str]
+    settings: Any, api_server: Optional[str]
 ) -> Dict[str, Any]:
-    if settings.AUTH is False and api_url is None:
-        api_url = prompt.Prompt.ask("\n[cyan]API[/] URL address")
-        settings.SERVER = api_url
-    elif settings.AUTH is False and api_url is not None:
-        settings.SERVER = api_url
+    if api_server is None:
+        api_server = prompt.Prompt.ask("\n[cyan]API[/] URL address")
+        settings.SERVER = api_server
```

Comment:
If we have used the `--config` option and `--api-server` and there is a server URL in the config file this means that the `--api-server` option will have a higher priority right?